work12/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="./style.css" type="text/css" />
+  <title>TerraceTechフロントエンドエンジニア養成所</title>
+</head>
+
+<body>
+  <div class="wrap" id="js-wrap">
+    <ul class="list" id="js-list"></ul>
+  </div>
+
+  <div id="js-button" class="button"><button type="submit">クリック</button></div>
```

Comment:
IMO
Is it possible to create buttons in js?

Reply:
@kanakonagiri 
thank you.
I created a click button with js.
After the DOM is generated, "document.addEventListener (" DOMContentLoaded ", function () {})" to get the click button element
using.

commit: 8d9f782